app/controllers/application_controller/report_downloads.rb

```diff
@@ -211,7 +211,7 @@ def set_summary_pdf_data
       :page_layout => "portrait",
       :page_size   => "us-letter",
       :run_date    => run_time.strftime("%m/%d/%y %l:%m %p %z"),
-      :title       => "#{klass} \"#{@record.name}\"".html_safe,
+      :title       => "#{klass} \"#{@record.respond_to?(:name) ? @record.name : @record.description}\"".html_safe,
```

Comment:
Can you please explain why is this extra complexity necessary?

Reply:
@skateman because `ChargebackRate` record does not have name field. Previously we did not have download buttons on the Chargeback Rates list view because we did not render gtl view toolbar on the screen.